ci/tools/merge_cuda_core_wheels.py

```diff
@@ -0,0 +1,189 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+#!/usr/bin/env python3
```

Comment:
nit: the shebang has to be the first line, IIUC.

Reply:
fixed in commit 20c5a99